environments/gym_adapter.py

```diff
@@ -29,7 +29,7 @@ class Environment:
     def __init__(self, cfg):
         self.gym_env_name = cfg.env_name
 
-        gym_env = gym.make(self.gym_env_name)
+        gym_env = gym.make(self.gym_env_name, new_step_api=True)
```

Comment:
Do we really need to introduce new input argument? since we plant to migrate to `gymnasium`

Reply:
To remove the current warning, yes. We can deal with the other changes when migrating to gymnasium.